tests/basictests.cpp

```diff
@@ -1949,6 +1957,11 @@ int main(int argc, char *argv[]) {
   if (simdjson::active_implementation->name() == "unsupported") {
     printf("unsupported CPU\n");
   }
+  // We want to know what we are testing.
+  std::cout << "Detected the best implementation for your machine: " << simdjson::active_implementation->name();
```

Comment:
Granted. And I would argue that all of our tests should print out their configuration.

(I am trying to avoid adding more commits to this PR because... too many already...)

Reply:
Totally fine. Nits is nits. Only thing I think I care about is your opinion on the test coverage of -307...308.

Reply:
Will fix.

Reply:
Fixed in a commit.